src/tests/functional/inference_engine/transformations/common_optimizations/dimension_tracking.cpp

```diff
@@ -238,6 +238,69 @@ TEST(TransformationTests, AutoBatch_FindBatch_NegativeTracking) {
     ASSERT_TRUE(!ov::DimensionTracker::get_label(out_shape[0])) << out_shape;
 }
 
+TEST(TransformationTests, AutoBatch_FindBatch_AutoBatch_LabelPropagation_DO_detachment) {
```

Comment:
there is an nraph builder helper with the DO that I use in the auto-batching tests, it is pretty much the same, maybe we shall reuse that to align